homeassistant/components/media_player/braviatv.py

```diff
@@ -132,15 +145,16 @@ def setup_bravia(config, pin, hass, add_devices_callback):
                 {host: {'pin': pin, 'host': host, 'mac': mac}}):
             _LOGGER.error('failed to save config file')
 
-        add_devices_callback([BraviaTVDevice(host, mac, name, pin)])
+        add_devices([BraviaTVDevice(host, mac, name, pin)])
 
 
 def request_configuration(config, hass, add_devices_callback):
     """Request configuration steps from the user."""
     host = config.get(CONF_HOST)
     name = config.get(CONF_NAME)
+
     if name is None:
-        name = "Sony Bravia"
+        name = DEFAULT_NAME
```

Comment:
This method is invoked after voluptuous - should already be populated - again, if not, use `config.get(CONF_NAME, DEFAULT_NAME)`